compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -183,7 +183,7 @@ tests Tools{damlc} = testGroup "Packaging" $
           , "a = ()"
           ]
         writeFileUTF8 (projDir </> "daml.yaml") $ unlines
-          [ "sdk-version: a-bad-sdk-version"
+          [ "sdk-version: 0.0.0-a-bad-sdk-version"
```

Comment:
Note that GHC package versions can only have numbers, so all non-number segments are removed (somewhere). If you want to test a "different" version, maybe use 0.0.13 for this one?

Reply:
This only tests that the sdk version used to compile is the one provided by DAML_SDK_VERSION - ghc packaging doesn't end up being an issue here, as that would be caused by `version:`.